docs/_includes/footer.html

```diff
@@ -3,10 +3,17 @@
 
     <ul class="icons">
         <li><a href="{{ site.github }}" class="icon circle fa-github"><span class="label">Github</span></a></li>
+        <li><a href="mailto:emodb-open-source@bazaarvoice.com" class="icon circle fa-envelope"><span class="label">Email</span></a></li>
+    </ul>
+
+
+    <ul class="copyright">
+        <li><a href="https://github.com/bazaarvoice/emodb/blob/master/Credits.md">Project Credits</a></li>
```

Comment:
I feel when we do the big push of porting the pages over, this would need to be updated.
I can open another issue and PR to do the documentation porting.